docs/user_manual/calculations.md

```diff
@@ -77,7 +77,9 @@ a faulty outcome instead of raising a singular matrix error.
 ### Power flow algorithms
 Two types of power flow algorithms are implemented in power-grid-model; iterative algorithms (Newton-Raphson / Iterative current) and linear algorithms (Linear / Linear current).
 Iterative methods are more accurate and should thus be selected when an accurate solution is required. Linear approximation methods are many times faster than the iterative methods, in tradeoff to accuracy. 
-They can be used where approximate solutions are acceptable. The table below can be used to pick the right algorithm. Below the table a more in depth explanation is given for each algorithm.
+They can be used where approximate solutions are acceptable. 
+Their accuracy is not explicitly calculated and may vary a lot. The user should have an intuition of their applicability based on the input grid configuration.
```

Comment:
The accuracy of linear approximation methods ...

(New paragraph, so unknown what their refers to)

Reply:
I know, thought I saw a white line in between yesterday. Must have seen it wrong then